### PR TITLE
EDSC-1233 fix ESI Service request bbox coordinates

### PR DESCRIPTION
**Pull request by macrouch**
these specific echoforms used `boundbox1` instead of `boundingbox` I copied the `find_by_xpath` call from Reverb to fix the issue

### CODE REVIEW

**Comment by mightynimble** [comment]
Add a test?

**Comment by macrouch** [comment]
I wasn't sure how to get an ordering test working. I'll keep trying

File: spec/features/data_access/service_options_with_boundingbox1_spec.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      expect(page).to have_checked_field('Enter bounding box')
+      fill_in 'Email Address', with: 'test@email.com'
+      fill_in 'North', with: '-63'
+      fill_in 'West', with: '-139'
+      fill_in 'East', with: '-100'
+      fill_in 'South', with: '-76'
+      wait_for_xhr
+
+      within '.access-item-actions' do
+        sleep 2
```
Comment (mightynimble) [inline spec/features/data_access/service_options_with_boundingbox1_spec.rb]: Why do we need to `sleep` here?

Comment (macrouch) [inline spec/features/data_access/service_options_with_boundingbox1_spec.rb]: I'm not 100% sure we do, I copied the majority of the test from spec/features/data_access/service_options_with_bad_bounding_box_spec.rb

**Pull request merged**